lib/compilers/ppci.js

```diff
@@ -0,0 +1,20 @@
+
+const Compile = require('../base-compiler'),
+    exec = require('../exec');
+
+function compilePpci(info, env) {
+    const compiler = new Compile(info, env);
+
+    compiler.exec = function (compiler, args, options) {
+        const python = 'python3';
+        const python_path = '/opt/compiler-explorer/' + compiler;
```

Comment:
I agree with @partouf . The whole path should be on the properties file.
Ideally, the final command looks something like `python -m /path/to/ppci.main.executable -arguments -needed -to -compile inputfilename.c` but I'm open to alternatives if you think that is not viable :)

Reply:
Okay, I'll put the whole path in the exe property.